hw/ip_templates/pwrmgr/dv/tb.sv.tpl

```diff
@@ -115,7 +115,14 @@ module tb;
     .strap_o    (pwrmgr_if.strap),
     .low_power_o(pwrmgr_if.low_power),
 
-    .rom_ctrl_i(pwrmgr_if.rom_ctrl),
+    .rom_ctrl_i({
+  % if NumRomInputs > 1:
+    // TODO: Properly model rom_ctrl if there are more than 1
+  % endif
+  % for i in range(NumRomInputs-1):
```

Comment:
I don't understand what this line is doing. I guess it covers for pwrmgr_if to have a single rom_ctrl, and that is the actual TODO. Eventually it will be an array. Could you clarify the TODO so this is more understandable and it is clearer what needs to be done?

Reply:
That one has be resolved to make an array in the interface.